astroplan/constraints.py

```diff
@@ -997,24 +998,90 @@ def observability_table(constraints, observer, targets, times=None,
     return tab
 
 
-def _rescale_minmax(vals, min_val, max_val):
-    """ Rescale altitude into an observability score."""
-    rescaled = (vals - min_val) / (max_val - min_val)
-    below = rescaled < 0
-    above = rescaled > 1
-    rescaled[below] = 0
-    rescaled[above] = 1
+def min_best_rescale(vals, min_val, max_val, greater_than=0, less_than=1):
```

Comment:
How about naming the last two arguments (which might be just one if you make the changes I'm about to discuss in the comments) as "`greater_score` or similar?  That makes it clearer that you're saying what the *output* is, rather than what input value is.

Similarly, perhaps you should use the word "score" in the docstring.  E.g. 
```
Rescales the input array ``vals`` to be a score (between zero and one), where ``min_val`` 
is the best and ``max_val`` is whatever ``greater_score`` is. 
```